packages/core/src/index.ts

```diff
@@ -110,7 +110,11 @@ export function buildSendMail(options: BuildSendMailOptions) {
     }
 
     if (forcePreview) {
-      log("💌 opening sendMail preview", mail);
+      const debugMail = {
+        ...mail,
+        html: "omitted",
+      };
+      log("💌 opening sendMail preview", debugMail);
```

Comment:
snuck in this thing to make this log more readable

Reply:
i think it is much better because you can pull up the html in the previewer view anyway